README.md

```
@@ -1,134 +1,259 @@
-### Apps and Packages
+### Web(사용자)
 
-- `docs`: a [Next.js](https://nextjs.org/) app
-- `web`: another [Next.js](https://nextjs.org/) app
-- `@repo/ui`: a stub React component library shared by both `web` and `docs` applications
-- `@repo/eslint-config`: `eslint` configurations (includes `eslint-config-next` and `eslint-config-prettier`)
-- `@repo/typescript-config`: `tsconfig.json`s used throughout the monorepo
+## 로그인
 
-Each package/app is 100% [TypeScript](https://www.typescriptlang.org/).
+<p align="center"> <img src="./source/login.png" width="300" /> </p>
```

markdownlint-cli2 (0.21.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)